processor/k8sprocessor/config.go

```diff
@@ -193,3 +197,8 @@ type PodAssociationConfig struct {
 	// e.g. ip, pod_uid, k8s.pod.ip
 	Name string `mapstructure:"name"`
 }
+
+// PodIgnoredConfig represent a list of Pod names to ignore
+type PodIgnoredConfig struct {
+	Names []string `mapstructure:"names"`
+}
```

Comment:
```suggestion
type PodIgnoredConfig []string
```

The config entry is already called `ignored_pod_names`, so having an additional `names` entry in the map as opposed to just making it a list seems overkill.